book/C-git-commands/1-git-commands.asc

```diff
@@ -234,124 +234,133 @@ Sa <<_git_branching#_create_new_branch>> ginamit namin ito kasama ang `--decorat
 Sa <<_distributed_git#_private_team>> at <<_git_tools#_commit_ranges>> aming tinalakay ang `branchA..branchB` na syntax para gamitin ang `git log` na utos para makita kung ano ang kaibahan sa isang katugong branch sa iba pang branch.
 Sa <<_git_tools#_commit_ranges>> malawak naming tinalakay ito.
 
-In <<_git_tools#_merge_log>> and <<_git_tools#_triple_dot>> we cover using the `branchA...branchB` format and the `--left-right` syntax to see what is in one branch or the other but not in both.
-In <<_git_tools#_merge_log>> we also look at how to use the `--merge` option to help with merge conflict debugging as well as using the `--cc` option to look at merge commit conflicts in your history.
+Sa <<_git_tools#_merge_log>> at <<_git_tools#_triple_dot>> tinalakay natin sa `branchA...branchB` format at ang `--left-right` na syntax para makita kung ano ang nasa isang branch o sa ibang branch pero hindi sa dalawa.
 
-In <<_git_reflog>> we use the `-g` option to view the Git reflog through this tool instead of doing branch traversal.
+Sa <<_git_tools#_merge_log>> tiningnan then natin kung paano gamitin ang `--merge` na opsyon para makatulong sa pag-debug ng merge conflict pati na rin ang `--cc` na opsyon para makita ang pag-merge sa mga conflict na commit sa iyong kasaysayan.
 
-In <<_git_tools#_searching>> we look at using the `-S` and `-L` options to do fairly sophisticated searches for something that happened historically in the code such as seeing the history of a function.
+Sa <<_git_reflog>> ginamit natin ang `-g` na opsyon para makita ang reflog ng Git sa pamamagitan ng tool na ito imbes sa paggawa ng traversal sa branch.
 
-In <<_git_tools#_signing_commits>> we see how to use `--show-signature` to add a validation string to each commit in the `git log` output based on if it was validly signed or not.
+Sa <<_git_tools#_searching>> ginamit natin ang `-S` at `-L` na mga opsyon para gumawa ng mas pinahusay na mga paghahanap para sa mga anumang pangyayari na pangkasaysayan sa code gaya ng pagtanaw sa kasaysayan ng isang function.
+
+Sa <<_git_tools#_signing_commits>> nakita natin kung paano gamitin ang `--show-signature` para magdagdag ng pagpapatibay na string sa bawat commit sa output ng `git log` nakabase dito kung ito ay wastong pag-sign o hindi.
 
 ==== git stash
 
-The `git stash` command is used to temporarily store uncommitted work in order to clean out your working directory without having to commit unfinished work on a branch.
+Ang `git stash` na utos ay ginamit para pansamantalang pag-imbak sa hindi na commit na trabaho upang luminis ang iyong gumaganang direktoryo na hindi kailangang mag-commit sa di tapos na trabaho sa isang branch.
```

Comment:
``pansamantalang pag-imbak sa hindi na commit`` to ``pansamantalang mag-imbak ng hindi na-commit``